src/api/ddoc/views.rst

```diff
@@ -914,3 +917,45 @@ Sending multiple queries to a view
             }
         ]
     }
+
+.. _api/ddoc/view/pagination:
+
+Pagination
+==========
+
+.. versionadded:: 4.0
+
+There is a support for token based pagination. The token based pagination is an
```

Comment:
[Stylistic] Avoid passive voice, and combine sentences to reduce repetitiveness.

```suggestion
CouchDB supports token-based pagination, which is an
```